app/src/main/res/layout/activity_interaction_result.xml

```diff
@@ -13,13 +14,33 @@
         android:layout_centerHorizontal="true"
         android:indeterminateTint="@color/colorPrimary"/>
 
-    <TextView
-        android:id="@+id/resultTest"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:gravity="center"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentEnd="true"
-        android:layout_centerVertical="true" />
+    <LinearLayout
```

Comment:
My recommendation is to change this LinearLayout to RelativeLayout.
The default orientation of LinearLayout is horizontal, Which means that the items that you put in LinearLayout stacks horizontally.

So say for example that the interaction result came out to be "Safe Status". We should show "@+id/safe" image with TextView and make "@+id/warning" image visibility to NONE. What I think will happen is that since this is LinearLayout, none of the objects will be in the same position. Instead, they will be stacked. This will cause two images showing in different position when the result is "Safe Status" and when the result is "Warning Status".
![image](https://user-images.githubusercontent.com/27392567/77247268-b6f80300-6c72-11ea-8228-b63bfb2f6a8e.png)
This image is to make my explanation more simple, not meaning that it will be exactly like that.

Reply:
fixed.